packages/fxa-auth-server/lib/oauth/db/index.js

```diff
@@ -61,7 +69,13 @@ class OauthDB {
       vals.expiresAt,
       vals.ttl
     );
-    if (POCKET_IDS.includes(hex(vals.clientId))) {
+    if (
+      JWT_ACCESS_TOKENS_ENABLED &&
+      JWT_ACCESS_TOKENS_CLIENT_IDS.has(vals.clientId) &&
+      token.ttl <= SHORT_ACCESS_TOKEN_TTL_IN_MS
```

Comment:
This will cause Firefox to make JWT access tokens for any sufficiently-short-lived token, regardless of scope. That seems fine and good to me (and could be helpful if we want to test whether other systems accept JWTs, e.g. the profile server) but I wanted to note it explicitly in case it wasn't intend. I don't recall whether we talked about limited by "oldsync" scope here or not.

Reply:
I originally had it for oldsync only, but we create a lot of 5 minute tokens for the login screen to fetch the avatar (which are already JWT tokens) so I decided to try it for all types. It should noticeably decrease our redis dataset.